AutomatedTesting/Gem/PythonTests/assetpipeline/ap_fixtures/bundler_batch_setup_fixture.py

```diff
@@ -158,7 +158,7 @@ def _append_arguments(self, cmd: List[str], kwargs: Dict, append_defaults: bool
                 else:
                     cmd.append(f"--{key}")
             if append_defaults:
-                cmd.append(f"--project-path={workspace.project}")
+                cmd.append(f"--project-path={os.path.join(workspace.paths.engine_root(), workspace.project)}")
```

Comment:
There is already a function to retrieve the absolute project path.

It is the `workspace.paths.project()` method

Reply:
Switched to that, I'm running Jenkins with it now

Reply:
All of the bundler periodic tests passed with that change